DSharpPlus.Test/Program.cs

```diff
@@ -13,19 +13,19 @@ public static void Main(string[] args)
 
         private async Task Run(string[] args)
         {
-            var cfg = TestBotConfig.Default;
+            var cfg = new TestBotConfig();
```

Comment:
I think this should be left at `var cfg = TestBotConfig.Default;`, since there are default values not covered by the constructor IIRC. Might be wrong though. Unless the config is updated appropriately.